google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/admin/v2/models/Cluster.java

```diff
@@ -156,6 +157,20 @@ public StorageType getStorageType() {
     return StorageType.fromProto(stateProto.getDefaultStorageType());
   }
 
+  /**
+   * Google Cloud Key Management Service (KMS) settings for a CMEK-protected Bigtable cluster.
```

Comment:
nit. Please add small blurb that it will return the full resource name (unlike all other identifiers, which are bare ids)

Reply:
Done.